internal/database/dbconn/open.go

```diff
@@ -376,8 +376,7 @@ func argsAsAttributes(ctx context.Context, _ otelsql.Method, _ string, args []dr
 			attrs[i] = attribute.StringSlice(key, strings)
 
 		default: // in case we miss anything
-			// To prevent very large attributes, render at most 128 characters.
-			attrs[i] = attribute.String(key, fmt.Sprintf("%.128v", v))
+			attrs[i] = attribute.String(key, fmt.Sprintf("%v", v))
```

Comment:
Maybe we should just log the type here?

```suggestion
			attrs[i] = attribute.String(key, fmt.Sprintf("%T", v))
```

Reply:
If we encounter something we want to see we should write a special handler for it

Reply:
makes sense to me, let's do that 👍